README.md

```diff
@@ -77,4 +89,6 @@ This repository contains helper scripts and commands to create meetup videos for
 
 * create the uploadable version
 
-        $ ffmpeg -i title.mp4 -i final2.mp4 -filter_complex "[0:0] [0:1] [1:0] [1:1] concat=n=2:v=1:a=1 [v] [a]" -map "[v]" -map "[a]" uploadable.mp4
+        $ ffmpeg -i title.mp4 -i final2.mp4 -filter_complex "[0:0] [0:1] [1:0] [1:1] concat=n=2:v=1:a=1 [v] [a]" -map "[v]" -map "[a]"  uploadable.mp4
+
+    Note: if you aren't worried about file size & want to avoid 100% all cpu core utilization, you can add `-preset ultrafast` to speed up the processing
```

Comment:
yeah, sure. I just wanted to highlight things which bothered me. I am absolutely fine with improvisation.